docs/content/tutorials/tutorial-kafka.md

```diff
@@ -37,149 +30,56 @@ Start a Kafka broker by running the following command in a new terminal:
-  ]
-  ```
-
-  ```json
-  "properties" : {
-       "task.partitions" : "1",
-       "task.replicants" : "1",
-       "topicPattern" : "pageviews"
-  }
-  ```
-
-Next, start Druid Kafka ingestion:
+In your Kafka directory, run the following command, where {PATH_TO_DRUID} is replaced by the path to the Druid directory:
 
 ```bash
-bin/tranquility kafka -configFile ../druid-#{DRUIDVERSION}/conf-quickstart/tranquility/kafka.json
+export KAFKA_OPTS="-Dfile.encoding=UTF-8"
+./bin/kafka-console-producer.sh --broker-list localhost:9092 --topic wikipedia < {PATH_TO_DRUID}/quickstart/wikipedia-2015-09-12-sampled.json
 ```
 
-- If your Tranquility server or Kafka is already running, stop it (CTRL-C) and
-start it up again.
-
-Finally, send some data to the Kafka topic. Let's start with these messages:
-
-```json
-{"time": "2000-01-01T00:00:00Z", "url": "/foo/bar", "user": "alice", "latencyMs": 32}
-{"time": "2000-01-01T00:00:00Z", "url": "/", "user": "bob", "latencyMs": 11}
-{"time": "2000-01-01T00:00:00Z", "url": "/foo/bar", "user": "bob", "latencyMs": 45}
-```
+The previous command posted sample events to the *wikipedia* Kafka topic which were then ingested into Druid by the Kafka indexing service. You're now ready to run some queries!
 
-Druid streaming ingestion requires relatively current messages (relative to a slack time controlled by the
-[windowPeriod](../ingestion/stream-ingestion.html#segmentgranularity-and-windowperiod) value), so you should
-replace `2000-01-01T00:00:00Z` in these messages with the current time in ISO8601 format. You can
-get this by running:
+## Querying your data
 
-```bash
-python -c 'import datetime; print(datetime.datetime.utcnow().strftime("%Y-%m-%dT%H:%M:%SZ"))'
-```
+After data is sent to the Kafka stream, it is immediately available for querying.
 
-Update the timestamps in the JSON above, then copy and paste these messages into this console
-producer and press enter:
+Please follow the [query tutorial](../tutorial/tutorial-query.html) to run some example queries on the newly loaded data.
 
-```bash
-./bin/kafka-console-producer.sh --broker-list localhost:9092 --topic pageviews
-```
+## Cleanup
 
-That's it, your data should now be in Druid. You can immediately query it using any of the
-[supported query methods](../querying/querying.html).
+If you wish to go through any of the other ingestion tutorials, you will need to shut down the cluster and reset the cluster state by removing the contents of the `var` directory under the druid package, as the other tutorials will write to the same "wikipedia" datasource.
 
 ## Further reading
 
-To read more about loading streams, see our [streaming ingestion documentation](../ingestion/stream-ingestion.html).
+For more information on loading data from Kafka streams, please see the [Druid Kafka indexing service documentation](http://druid.io/docs/{{druidVersion}}/development/extensions-core/kafka-ingestion.html)..
```

Comment:
There's an extra period at the end of this line

Reply:
Fixed